python/buku/Portfile

```diff
@@ -25,13 +25,15 @@ long_description    buku is a powerful bookmark manager written in \
                     buku is too busy to track you: no hidden history, \
                     obsolete records, usage analytics or homing.
 
-categories          www
+homepage            https://github.com/jarun/buku
+
+categories-append   www
 platforms           {darwin any}
 supported_archs     noarch
 maintainers         {isi.edu:calvin @cardi} openmaintainer
 license             GPL-3
 
-python.default_version  39
+python.versions     38 39 310 311 312
```

Comment:
This doesn't do anything useful. As per the docs, subports are only created for module ports, not apps. Setting `python.default_version` is required for apps (because leaving it as a default value that will change in future makes the build non-reproducible). If you want to offer a choice of python versions, you need to create variants that change `python.default_version`. But it's fine not to do that and just set `python.default_version` to the current preferred version.

Reply:
Thanks for the review: I've opened https://github.com/macports/macports-ports/pull/23442 to set `python.default_version` to 3.12.

Can users set and override `python.default_version`? Thinking about the case if users set their preferred version to something that a Python application doesn't support (e.g., buku does not support 3.7), I think it would be a better experience to output a message ("app requires Python >= 3.8") rather than a build/install error.

Reply:
While it is possible to override any option on the command line, that's a tool for development and debugging and not a supported configuration method.

Reply:
Good to know, thank you!